### ISSUE TITLE
您好，请问您的验证集怎么是这个，要自己下载吗，

### ISSUE THREAD

**Issue by zycc853** [MasLiang/Learning-without-Forgetting-using-Pytorch#4]
valloader = torch.utils.data.DataLoader(
        datasets.ImageFolder(root='~/Imagenet2012/ILSVRC2012_img_val', transform=tfs.Compose([
            tfs.Resize(256),
            tfs.CenterCrop(224),
            tfs.ToTensor(),
            normalize,
            ])),

**Comment by MasLiang** [comment]
It depends on you. You can use any dataset.

**Comment by zycc853** [comment]
好的，谢谢



---原始邮件---
发件人: "Jiawei ***@***.***&gt;
发送时间: 2021年8月28日(周六) 中午11:31
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [MasLiang/Learning-without-Forgetting-using-Pytorch] 您好，请问您的验证集怎么是这个，要自己下载吗， (#4)




 
It depends on you. You can use any dataset.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
